### PR TITLE
OCPBUGS-36727#revising wrong command

### PR DESCRIPTION
**Pull request by brendan-daly-red-hat**
Versions: 
4.13+

Issue:
https://issues.redhat.com/browse/OCPBUGS-36727

Link to docs preview:
https://78940--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/configuring-iam-ibm-cloud#cco-ccoctl-configuring_configuring-iam-ibm-cloud

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by brendan-daly-red-hat** [comment]
@gpei, PTAL at this update. Thanks

**Review by lahinson — COMMENTED** [review]
LGTM

File: modules/cco-ccoctl-configuring.adoc [hunk 1]
```diff
@@ -371,14 +371,14 @@ $ chmod 775 ccoctl.<rhel_version>
 
 .Verification
 
-* To verify that `ccoctl` is ready to use, display the help file by running the following command:
+* To verify that `ccoctl` is ready to use, display the help file. Use a relative file name when you run the command, for example:
 +
 [source,terminal]
 ----
-$ ccoctl --help
+$ ./ccoctl.rhel9
```
Comment (jeana-redhat) [inline modules/cco-ccoctl-configuring.adoc:378]: @gpei is the `--help` not required to get the help output?

Comment (gpei) [reply modules/cco-ccoctl-configuring.adoc:378]: @jeana-redhat  both of the cases could work on my side.
```
[root@gpei-test-rhel9 ~]# ./ccoctl.rhel9 
OpenShift credentials provisioning tool

Usage:
  ccoctl [command]

Available Commands:
  alibabacloud Manage credentials objects for alibaba cloud
  aws          Manage credentials objects for AWS cloud
  azure        Manage credentials objects for Azure
  completion   Generate the autocompletion script for the specified shell
  gcp          Manage credentials objects for Google cloud
  help         Help about any command
  ibmcloud     Manage credentials objects for IBM Cloud
  nutanix      Manage credentials objects for Nutanix

Flags:
  -h, --help   help for ccoctl

Use "ccoctl [command] --help" for more information about a command.
[root@gpei-test-rhel9 ~]# ./ccoctl.rhel9 --help
OpenShift credentials provisioning tool

Usage:
  ccoctl [command]

Available Commands:
  alibabacloud Manage credentials objects for alibaba cloud
  aws          Manage credentials objects for AWS cloud
  azure        Manage credentials objects for Azure
  completion   Generate the autocompletion script for the specified shell
  gcp          Manage credentials objects for Google cloud
  help         Help about any command
  ibmcloud     Manage credentials objects for IBM Cloud
  nutanix      Manage credentials objects for Nutanix

Flags:
  -h, --help   help for ccoctl

Use "ccoctl [command] --help" for more information about a command.
```

Comment (jeana-redhat) [reply modules/cco-ccoctl-configuring.adoc:378]: Ok perfect thank you for confirming!

**Pull request merged**